feat(kno-12316): document dynamic audiences (#1417)
knocklabs/docs: content/concepts/conditions.mdx

    | Example condition | Evaluation |
    | --- | --- |
    | `age is greater_than 18` where `age` is `21` | true |
    | `age is greater_than 18` where `age` is `18` | false |

  </Accordion>
  <Accordion title="`greater_than_or_equal_to`">
    True if the variable is greater than or equal to the argument (`>=`).

    | Example condition | Evaluation |
    | --- | --- |
    | `age is greater_than_or_equal_to 18` where `age` is `18` | true |
    | `age is greater_than_or_equal_to 18` where `age` is `17` | false |

  </Accordion>
  <Accordion title="`less_than`">
    True if the variable is less than the argument (`<`).

    | Example condition | Evaluation |
    | --- | --- |
    | `score is less_than 50` where `score` is `49` | true |
    | `score is less_than 50` where `score` is `50` | false |

  </Accordion>
  <Accordion title="`less_than_or_equal_to`">
    True if the variable is less than or equal to the argument (`<=`).

    | Example condition | Evaluation |
    | --- | --- |
    | `score is less_than_or_equal_to 50` where `score` is `50` | true |
    | `score is less_than_or_equal_to 50` where `score` is `51` | false |

  </Accordion>
  <Accordion title="`contains`">

>     True if the argument appears in the variable. For text, checks whether the argument is a substring of the variable. For lists, checks whether the argument is an item in the list. When both values are lists, true if they share any items in common. To check whether one list fully contains the other, use `contains_all`.

Reviewer: For tex, if the argument is a list, will check if the text is fully either argument.

ie: 
"role" "contains" ["engineer", "admin"] 
if role is "engineer", true
if role is "admin", true
if role is "president", false

Author: I _think_ this could be covered here: 
> For lists, checks whether the argument is an item in the list.

but wonder how we can rewrite to be easier to understand...
```suggestion
True if the argument appears in the variable. 
- When one value is text and one is a list, true if the text is an item in the list. 
- When both values are text, true if the argument is a substring of the variable. 
- When both values are lists, true if they share any items in common. To check whether one list fully contains the other, use `contains_all`.

```

Reviewer: yeah i thihnk that works!
